.github/workflows/docs-checks.yml

```diff
@@ -43,14 +43,12 @@ jobs:
       - name: Install dependencies
         env:
           FREEZE_REQUIREMENTS: 1
-          PACKAGE_NAME: ${{ matrix.pkg }}
```

Comment:
Why remove this? I thought this is required to `pip install -e .`

Reply:
it isn't. If it isn't set, then it installs everything in editable mode (`pytorch_lightning`, `lightning_app` and `lightning_lite`)

Reply:
which is exactly what we want

Reply:
There seems to be some strange interference, so this is a bypass for unblocking development...

Reply:
The problem is, that `app` docs are referring to `pytorch` portion of the package and therefore, is trying to import it and since it doesn't have all the dependencies, it fails